consensus/propagation/reactor.go

```diff
@@ -111,7 +110,8 @@ func NewReactor(
 	// start the catchup routine
 	go func() {
 		// TODO dynamically set the ticker depending on how many blocks are missing
-		ticker := time.NewTicker(RetryTime)
+		retryTime := time.Second * 6
+		ticker := time.NewTicker(retryTime)
```

Comment:
this used to break the e2e test, but now its not

the reason for changing this was that there was a data race with the old global var

do we still need this @rach-id ? in theory, no correct since we're calling this at the end of every block?

Reply:
interesting. Can we not remove it in this PR and create an issue for it? I would like to further chew on it because the propagation reactor is missing some edge cases that I can't find and leads the e2e to fail occasionally.

In fact, after stopping the legacy gossip routine, I am wondering if that affects the votes and we're not receiving 2/3rds and not adding the commitment, therefore not catching up... But I'm not sure